src/Microsoft.DocAsCode.Build.JavaScriptReference/BuildInputs/MemberType.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.JavaScriptReference
+{
+    using System;
+
+    [Serializable]
+    public enum MemberType
+    {
+        Default,
+        Toc,
+        Package,
+        Class,
+        Interface,
+        Enum,
```

Comment:
Nice catch... although it's different from C#'s namespace, it do has this type... will add it back.

---
In reply to: [93221231](https://github.com/dotnet/docfx/pull/1016#discussion_r93221231) [](ancestors = 93221231,93191129,93179738)